kentikapi/client.go

```diff
@@ -161,8 +203,28 @@ func makeConnForGRPC(c Config, hostPort string) (grpc.ClientConnInterface, error
 	)
 }
 
-func makeTLSOption(c Config) grpc.DialOption {
-	if c.DisableTLS {
+func makeGRPCHostPort(apiURL string) (grpcHostPort string, tlsEnabled bool, err error) {
```

Comment:
I see that this branch includes suggestions from previous PR and I see larger diff than optimal . Please rebase the branch onto dev_grpc.
```
 * f75b0ab  (upstream/dev_grpc) TestClient_GetAgentWithRetries erro timeout fix (#92) (10 hours ago) <Jan Dziurzyński>
* 2e6fa67  Modify cloud export properties field to emulate union (#89) (14 hours ago) <Daniel Furman>
* 7854f6b  Fix payloads apiv6 (#85) (14 hours ago) <Jan Dziurzyński>
* e13b3c5  Implemeted suggestions from review (#86) (14 hours ago) <Jan Dziurzyński>
| * 66542ed  (HEAD -> client_functional_options) Client config functional options (5 days ago) <Jan Dziurzynski>
| * ae71411  Implemented suggetsios from review (5 days ago) <Jan Dziurzynski>
|/  
* 44fc3a9  (dev_grpc) RequiredFields structs for constructors of models (#82) (7 days ago) <Jan Dziurzyński>
```

Reply:
Done